resources/web/wwi/FloatingRobotWindow.js

```diff
@@ -1,12 +1,18 @@
 import FloatingWindow from './FloatingWindow.js';
 
 export default class FloatingRobotWindow extends FloatingWindow {
-  constructor(parentNode, name, url, windowName) {
+  constructor(parentNode, name, url, windowName, mainWindow) {
     super(parentNode, name, url);
+    this.isMainWindow = mainWindow ? mainWindow : false;
     this.window = windowName;
-    this.headerText.innerHTML = 'Robot Window: ' + name;
+    this.headerText.innerHTML = mainWindow ? 'World Info' : 'Robot Window: ' + name;
```

Comment:
```suggestion
    this.headerText.innerHTML = mainWindow ? 'World Info ' : 'Robot Window: ' + name;
```

Reply:
I don't believe a space is necessary here as we do not include the `name` when using the `World Info` title. However we can if you want.